pallets/ethereum-xcm/src/tests/v3/eip7702.rs

```diff
@@ -139,16 +140,18 @@ fn test_transact_xcm_evm_call_works() {
 	let bob = &pairs[1];
 
 	ext.execute_with(|| {
-		let t = EIP7702UnsignedTransaction {
-			nonce: U256::zero(),
-			max_priority_fee_per_gas: U256::one(),
-			max_fee_per_gas: U256::one(),
+		let t = EthereumXcmTransactionV3 {
 			gas_limit: U256::from(0x100000),
-			destination: ethereum::TransactionAction::Create,
+			action: ethereum::TransactionAction::Create,
 			value: U256::zero(),
-			data: hex::decode(CONTRACT).unwrap(),
+			input: hex::decode(CONTRACT).unwrap().try_into().unwrap(),
+			access_list: None,
+			authorization_list: vec![].into(),
 		}
-		.sign(&alice.private_key, None, Vec::new());
+		.into_transaction(U256::zero(), Default::default(), true)
+		.unwrap();
+		// We do not support EIP7702 transactions from XCM
```

Comment:
Note: maybe we should document this in the `EthereumXcmTransactionV3::authorization_list` docs.